src/protagonist/API/Features/Image/Validation/HydraImageValidator.cs

```diff
@@ -36,7 +36,11 @@ public HydraImageValidator(IOptions<ApiSettings> apiSettings)
             .When(_ => !apiSettings.Value.DeliveryChannelsEnabled)
             .WithMessage("Delivery channels are disabled");
         
-        RuleForEach(a => a.DeliveryChannels)
+        RuleFor(a => a.ModelId).Must(id => id.IndexOfAny(apiSettings.Value.RestrictedAssetIdCharacters.ToCharArray()) == -1)

```

Comment:
Is it worth storing `RestrictedAssetIdCharacters` as a `char[]` if it's going to be converted to that for each check?